d3-selection/index.d.ts

```diff
@@ -845,7 +878,7 @@ interface Selection<GElement extends BaseType, Datum, PElement extends BaseType,
     /**
      * Return the first (non-null) element in this selection. If the selection is empty, returns null.
      */
-    node(): GElement | null;
+    node(): GElement;
```

Comment:
Convinced. I was just waiting for someone to say **bikesheding** :wink: I will make the change as part of the changes trigger by going to TS 2.2. with new es6 lib definitions.